src/compute/src/logging/compute.rs

```diff
@@ -398,6 +406,29 @@ impl<A: Allocate> DemuxState<A> {
     }
 }
 
+/// State tracked for each dataflow export.
+struct ExportState {
+    /// The ID of the dataflow maintaining this export.
+    dataflow_id: usize,
+    /// Imports feeding this export, and their frontier delay tracking state.
+    imports: BTreeMap<GlobalId, FrontierDelayState>,
+    /// Number of errors in this export.
+    ///
+    /// We keep this a signed integer to not be tripped up by negative total error counts. Seeing
+    /// negative counts would be a bug, but we still want to handle that gracefully.
```

Comment:
I think it's a bug if the _overall_ count across all workers is negative, but the per-worker count is fine to be negative. Only after a keyed exchange, the local counts are guaranteed to be positive.

Reply:
Ah, good point! I even saw that in the tests and amended the `mz_compute_error_counts` view accordingly. Will adjust the comment.